docs/getting-started/videos/microfrontend-composer.mdx

```diff
@@ -1,7 +1,7 @@
 ---
-id: backoffice
```

Comment:
You need to write the redirect about that.
You will add the redirect on the PR: #793